dimod/views/quadratic.py

```diff
@@ -352,6 +352,10 @@ def variables(self) -> Collection[Variable]:
     def add_linear(self, v: Variable, bias: Bias):
         raise NotImplementedError
 
+    @abc.abstractmethod
```

Comment:
That makes sense. However, shouldn't all the subclasses have this method? They do now. There are also `get_quadratic` `set_quadratic` etc. Let me know if that affects your comment otherwise I will proceed with your current solution!

Reply:
[get_quadratic()](https://github.com/dwavesystems/dimod/blob/323115f8bd45e0a9b7f76dcaa1af1b2e06983374/dimod/views/quadratic.py#L363-L365) and [set_quadratic()](https://github.com/dwavesystems/dimod/blob/323115f8bd45e0a9b7f76dcaa1af1b2e06983374/dimod/views/quadratic.py#L400-L402) are already abstract methods, so they are guaranteed to exist.

It's possible, although extremely unlikely, that some customer somewhere has their own implemented subclass that doesn't implemented `add_quadratic()` and so we would be breaking their code by suddenly requiring it... But yeah, extremely unlikely. IMO what you've done is fine, let's just note it in the release notes to cover our bases.